Jumpscale/builder/db/BuilderRedis.py

```diff
@@ -42,1215 +26,24 @@ def build(self, reset=False, start=False):
             rm -f /usr/local/bin/redis-server
             rm -f /usr/local/bin/redis-cli
             """
-            C = j.builder.tools.replace(C)
-            C = j.core.tools.text_replace(C)
-            j.sal.process.execute(C)
+            self._execute(C)
```

Comment:
it's better to use builder tools to perform these actions than executing it as bash script